pkg/tracking/store/sql/runs_internal_test.go

```diff
@@ -1,4 +1,3 @@
-//nolint:ireturn
```

Comment:
Please revert this as well. This must be the reason the linter is triggered on CI.